sw/device/tock/boards/opentitan/Cargo_local.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
Is there a way that this can be a symlink by default so that a person could just come in and `cargo build` without having to invoke meson?